tests/test_app/Plugin/PluginNamespaced/MyNamespacedPlugin/src/Plugin.php

```diff
@@ -0,0 +1,10 @@
+<?php
+declare(strict_types=1);
+
+namespace PluginNamespaced\MyNamespacedPlugin;
+
+use Cake\Core\BasePlugin;
+
+class Plugin extends BasePlugin
```

Comment:
Isn't there an already namespaced plugin which can be used instead of creating an additional test plugin?

Reply:
The company one. but I would rather migrate to a more clearly "named" one if possible.
As it wasnt clear from the start that it already exists as such.